### PR TITLE
feat: pickup item fixer

### PR DESCRIPTION
**Pull request by m1sk9**

### CODE REVIEW

File: src/main/java/net/azisaba/itemFixer/listeners/FixItemOnPickupListener.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+        if(eveEntity.getType() != EntityType.PLAYER) {
+            return;
+        }
+        if(eveEntity.hasPermission("itemfixer.exempt.fix_on_pickup")) {
+            return;
+        }
+
+        ItemStack item = e.getItem().getItemStack();
+        plugin.fixSingle(eveEntity.getName(), item);
```
Comment (acrylic-style) [inline src/main/java/net/azisaba/itemFixer/listeners/FixItemOnPickupListener.java]:
```suggestion
        plugin.fixSingle(eveEntity.getName() + " picked up an item", item);
```

File: src/main/java/net/azisaba/itemFixer/listeners/FixItemOnPickupListener.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+    public FixItemOnPickupListener(ItemFixer plugin) {
+        this.plugin = plugin;
+    }
+
+    @EventHandler(priority = EventPriority.HIGH)
+    public void onPickup(EntityPickupItemEvent e) {
+        Entity eveEntity = e.getEntity();
+
+        if(eveEntity.getType() != EntityType.PLAYER) {
```
Comment (acrylic-style) [inline src/main/java/net/azisaba/itemFixer/listeners/FixItemOnPickupListener.java]: add space between if and parentheses

File: src/main/java/net/azisaba/itemFixer/listeners/FixItemOnPickupListener.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+            return;
+        }
+        if(eveEntity.hasPermission("itemfixer.exempt.fix_on_pickup")) {
```
Comment (acrylic-style) [inline src/main/java/net/azisaba/itemFixer/listeners/FixItemOnPickupListener.java]: add space between if and parentheses

**Review by acrylic-style — CHANGES_REQUESTED** [review]
Did you test this?

**Comment by m1sk9** [comment]
> Did you test this?

I did a brain test.

**Comment by m1sk9** [comment]
I'm kidding. 
It's working.

**Review by acrylic-style — APPROVED** [review]

**Pull request merged**